src/System.Threading.Thread/tests/ThreadTests.cs

```diff
@@ -506,12 +506,34 @@ public static void ManagedThreadIdTest()
         [Fact]
         public static void NameTest()
         {
-            var t = new Thread(() => { });
+            string name = "a";
```

Comment:
Nit: to help avoid any issues, could we make this random? e.g. `string name = Guid.NewGuid().ToString("N");`?

Reply:
Sure will do